_templates/template.md

```diff
@@ -1,106 +1,228 @@
 ---
-# Jekyll 'Front Matter' goes here. Most are set by default, and should NOT be
-# overwritten except in special circumstances. 
-# You should set the date the article was last updated like this:
-date: 2020-05-11 # YYYY-MM-DD
-# This will be displayed at the bottom of the article
-# You should set the article's title:
-title: Title goes here
-# The 'title' is automatically displayed at the top of the page
-# and used in other parts of the site.
+date: {}
+title: NLP for robotics
+published: true
 ---
-This template acts as a tutorial on writing articles for the Robotics Knowledgebase. In it we will cover article structure, basic syntax, and other useful hints. Every tutorial and article should start with a proper introduction.
+# Introduction
```

Comment:
The guidelines clearly state no Markdown or HTML in the first paragraph/100 words. This becomes the page description, and Jekyll will include everything verbatim. I'd recommend dropping `# Introduction` and `## Overview` and keeping the first paragraph as-is.

Reply:
@nevalsar